.gitignore

```diff
@@ -1 +1,4 @@
 /node_modules/
+/esm/
+/cjs/
+/umd/
```

Comment:
Not sure about this no-final-newlines (here and in `.npmrc`) -  but just a nit.

Reply:
You're right. Fixed :+1: